docs/user-guides/deployment.md

```diff
@@ -0,0 +1,245 @@
+# Deployment
+
+There are two major ways for the deployment of the Vald cluster ways: Using the Helm command with `values.yaml` or `vald-helm-operator`.
+
+- Using Helm command
+    - Easy to deploy
+    - Need to Helm command when updating configuration
```

Comment:
user need to add helm repository by their selves

Reply:
and also helm command can edit vald configuration value when user execute helm command with inlining